repspark_at/actions/setup_class.py

```diff
@@ -0,0 +1,22 @@
+from appium import webdriver
+
+class setUpClass('???????????????'):
```

Comment:
Please take a look at the following example. Hope, it'll clarify the task for you.
```python
import unittest


class Test(unittest.TestCase):
    @classmethod
    def setUpClass(cls):
        cls.foo = 'foo'
        print("set class: {f}".format(f=cls.foo))

    @classmethod
    def tearDownClass(cls):
        cls.foo = None
        print("destroy class: {f}".format(f=cls.foo))


class Test1(Test):
    def test_1(self):
        print('{f}: test 1'.format(f=self.foo))

    def test_2(self):
        print('{f}: test 2'.format(f=self.foo))


if __name__ == '__main__':
    unittest.main()
```